script/legacy/legacy_db.php

```diff
@@ -49,7 +49,49 @@ function DB_LoadKeyboardsForLanguage($id) {
     $stmt->close();
     return $data;
   }
-      
+
+  function DB_LoadAllKeyboardLanguagesByLanguage($id) {
+    $stmt = new_query('CALL sp_legacy10_languages_for_keyboards_for_language(?)');
```

Comment:
Do you want to check $id? like line 67?
```
if(empty($id)) $id = null;
```

Reply:
`$id` shouldn't be null if you look at how this is used. It's not ideal code but it's legacy...